docs/src/modules/java/pages/views.adoc

```diff
@@ -653,3 +653,76 @@ To get the total number of results that will be returned over all pages, use `CO
 ----
 SELECT * AS customers, COUNT(*) AS total, has_more() AS more FROM customers LIMIT 10
 ----
+
+=== Paging
+
+Reading through a query result one "page" at a time rather than returning the entire result at once is possible in two ways, with a count based offset or a token based offset.
```

Comment:
```suggestion
Splitting a query result into one "page" at a time rather than returning the entire result at once is possible in two ways:
* with a count based offset or 
* a token based offset.
```